build/yamls/elk-flow-collector/elk-flow-collector.yml

```diff
@@ -188,14 +190,18 @@ spec:
   selector:
     app: logstash
   ports:
+    - port: 4738
```

Comment:
I hope this is not an officially registered port for any other protocol.

Reply:
It's not as I know. reference: https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers

Reply:
The wikipedia page is not really up-to-date: https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=4738
I don't knot if it matters though

Reply:
Thanks for the info. Updated the port to 4736 since it is not assigned yet https://www.adminsub.net/tcp-udp-port-finder/4736

Reply:
Thanks Yiou.
I was thinking of a scenario if ELK flow collector is placed outside in another cluster and exposed as a nodeport service.